packages/wallets/src/features/cashier/constants/constants.ts

```diff
@@ -49,49 +49,122 @@ export const walletIconMapper = {
     },
 } as const;
 
-export const appIconMapper = {
-    binary: {
-        dark: 'IcWalletOptionsDark',
-        light: 'IcWalletOptionsLight',
+export const LandingCompanyDetails = {
```

Comment:
Pls type this from the API. Can refer to types in wallets

Reply:
Updated